src/services/celebration-service.ts

```diff
@@ -56,7 +56,7 @@ export class CelebrationService {
 
                     // Get the blacklist data for this guild
                     let blacklistData = guildCelebrationData.blacklistedMembers.map(
```

Comment:
You might want to rename `blacklistedMembers` to just `blacklist` or something

Reply:
Done